ui/src/components/Footer/index.tsx

```diff
@@ -19,34 +19,51 @@
 
 import React from 'react';
 import { Container } from 'react-bootstrap';
-import { Trans } from 'react-i18next';
+import { Trans, useTranslation } from 'react-i18next';
 
 import dayjs from 'dayjs';
 
 import { siteInfoStore } from '@/stores';
 
-const Index = () => {
+const Footer = () => {
+  const { t } = useTranslation('translation', { keyPrefix: 'footer' }); // Scoped translations for footer
   const fullYear = dayjs().format('YYYY');
   const siteName = siteInfoStore((state) => state.siteInfo.name);
   const cc = `${fullYear} ${siteName}`;
+
   return (
     <footer className="bg-light">
       <Container className="py-3">
         <p className="text-center mb-0 small text-secondary">
-          <Trans i18nKey="footer.build_on" values={{ cc }}>
-            Powered by
-            {/* eslint-disable-next-line react/jsx-no-target-blank */}
-            <a href="https://answer.apache.org" target="_blank">
+          <a
```

Comment:
1. Use the Link component of react-router-dom to replace the a tag.
2. Remove target and rel attributes.